dtrace_probe.cc

```diff
@@ -41,16 +41,18 @@ namespace node {
   NAN_METHOD(DTraceProbe::Fire) {
     Nan::HandleScope scope;
     DTraceProbe *pd = Nan::ObjectWrap::Unwrap<DTraceProbe>(info.Holder());
-    info.GetReturnValue().Set(pd->_fire(info[0]));
+    info.GetReturnValue().Set(pd->_fire(info, 0));
   }
 
-  v8::Local<Value> DTraceProbe::_fire(v8::Local<v8::Value> argsfn) {
+  v8::Local<Value> DTraceProbe::_fire(Nan::NAN_METHOD_ARGS_TYPE argsinfo, int fnidx) {
```

Comment:
Good point. I'll look into reworking the change to collect the arguments before calling `_fire`.